### ISSUE TITLE
Criar versão com SQLAlchemy

### ISSUE THREAD

**Issue by rochacbruno** [rochacbruno/talkshow#18]
Após a maioria das issues de estrutura serem resolvidas a idéia é criar uma versão do mesmo app utilizando SQLAlchemy + SQLite. em uma nova branch.

**Comment by Caaddss** [comment]
tudo igual, só o banco diferente? certo?

**Comment by rochacbruno** [comment]
@Caaddss sim! tem um exemplo de uso do SQLAlchemy aqui https://github.com/rochacbruno/flask_admin_example/

A idéia é ter uma nova branch chamada `sqlalchemy` (baseada na branch `extended`) ou se possivel fazer com que seja na mesma branch `extended` mas com uma chave de configuração para mudar as estrategias de banco de dados.

**Comment by Caaddss** [comment]
@rochacbruno ahh beleza, primeiro vou fazer em uma branch separada, se não tiver problema

**Comment by tiagovizoto** [comment]
@Caaddss está trabalhando nisso?

**Comment by Caaddss** [comment]
estou sim @tiagovizoto

**Comment by Riverfount** [comment]
@Caaddss e @tiagovizoto como está o desenvolvimento dessa issue? Se pararam posso assumir! ;-)

**Comment by tiagovizoto** [comment]
@Caaddss informou que está trabalhando nisso. Mas realmente nem sei como está. 
@Riverfount , nem possui uma branch no Repositório do @Caaddss  para essa issue.

Preciso da camiseta
